ref/net46/Microsoft.Build/Microsoft.Build.cs

```diff
@@ -1005,6 +1008,7 @@ public enum BuildRequestDataFlags
         IgnoreExistingProjectState = 4,
         None = 0,
         ProvideProjectStateAfterBuild = 2,
+        ProvideSubsetOfStateAfterBuild = 32,
```

Comment:
Should we require this, or infer it when you pass a `RequestedProjectState`? Right now it's not really distinct.